pyspectral/utils.py

```diff
@@ -94,6 +96,7 @@
 ATM_CORRECTION_LUT_VERSION['rayleigh_only'] = {'version': 'v1.0.1',
                                                'filename': 'PYSPECTRAL_ATM_CORR_LUT_RO'}
 
+#: Aerosol types available as downloadable LUTs for rayleigh correction
```

Comment:
Could we skip the inline comment is we changed to:
`DOWNLOADABLE_AEROSOL_TYPES_FOR_ATM_CORRECTION`?

Otherwise if you still want the inline comment, change "rayleigh" to "atmospheric". Rayleigh scattering is happening in atmospheric molecules and atoms. Aerosols are mainly absorbing in the visible range. When we speak about rayleigh correction it is the substraction of the enhanced scattering due to atm molecules and atoms. Correcting for aerosol absorption is another atm correction.

Reply:
This is considered a "docstring" by sphinx. It was needed so that I could link to this list from the docstring in the related function. If I didn't have this comment here then the constant isn't included in the sphinx docs at all so users can't know what aerosol types are available unless they go into the source code.